server.js

```diff
@@ -199,11 +199,11 @@ app.get('/api/sms/', function(req, res) {
 app.get('/api/sms', function(req, res) {
   auth.check(req, 'can_access_gateway_api', null, function(err) {
     if (err) {
-      return serverUtils.error(err, req, res);
+      return serverUtils.error(err, req, res, true);
```

Comment:
It's definitely weird. I got rid of it for these calls. The documentation implies I should have it, but looking at the code it's not necessary (IMO in any situation).